vendor/QmXvNwFNR7CQJ1J5oJnjS4oAFv2ai6v9JZBkTBbJp6wRQf/go-ipfs-gateway/gateway_handler.go

```diff
@@ -21,6 +21,8 @@ import (
 	path "github.com/ipfs/go-ipfs/path"
 	"github.com/ipfs/go-ipfs/routing"
 	uio "github.com/ipfs/go-ipfs/unixfs/io"
+
+	logging "github.com/rht/go-ipfs-gateway/vendor/QmQg1J6vikuXF9oDvm4wpdeAUvvkVEKW1EYDw9HhTMnP2b/go-log"
```

Comment:
gx install works, but we need to decide on a final pathing format for gx before i'm comfortable pushing it all the way into go-ipfs. 

ref: https://github.com/whyrusleeping/gx/issues/21